plugins/janus_videoroom.c

```diff
@@ -1706,15 +1616,13 @@ static guint32 janus_videoroom_rtp_forwarder_add_helper(janus_videoroom_publishe
 	if(!is_data && srtp_suite > 0 && srtp_crypto != NULL) {
 		/* First of all, let's check if there's already an RTP forwarder with
 		 * the same SRTP context: make sure SSRC and pt are the same too */
-		char media[10];
-		memset(media, 0, sizeof(media));
+		char media[10] = {0};
```

Comment:
Does this set all the bytes in the char array to 0?

Reply:
yes it worked in c++

Reply:
C99 Standard 6.7.8.21
> If there are fewer initializers in a brace-enclosed list than there are elements or members of an aggregate, or fewer characters in a string literal used to initialize an array of known size than there are elements in the array, the remainder of the aggregate shall be initialized implicitly the same as objects that have static storage duration.

So yes, it's valid.

Reply:
Thanks for clarifying!